Common/TableProducer/PID/pidTOFBase.cxx

```diff
@@ -60,8 +60,8 @@ struct tofSignal {
       LOG(info) << "Table TOFSignal enabled!";
     }
   }
-  using Trks = o2::soa::Join<aod::Tracks, aod::TracksExtra>;
-  void process(Trks const& tracks)
+  using Trks = o2::soa::Join<aod::TracksIU, aod::TracksExtra>;
```

Comment:
@jezwilkinson with this PR there will be two process functions, one that works with Run3 and Run2 and one that works only with Run3 (but faster) what do you think?